scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -162,6 +163,8 @@ private void releaseNodes(InternalJob job, NodeSet nodeSet) throws RMProxyCreati
      * @return the number of tasks that have been started
      */
     public int schedule() {
+        schedulingMainLoopTimingLogger = new SchedulingMainLoopTimingLogger(Logger.getLogger(SchedulingMethodImpl.class));
```

Comment:
with the change mentionned above, you can use new SchedulingMainLoopTimingLogger(logger);